src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -722,22 +1373,62 @@ namespace Opm {
         }
     }
 
-    const ecl_grid_type * EclipseGrid::c_ptr() const {
-        return m_grid.get();
-    }
+    bool EclipseGrid::equal(const EclipseGrid& other) const {
 
+        //double reltol = 1.0e-6;
 
-    bool EclipseGrid::equal(const EclipseGrid& other) const {
-        bool status = (m_pinch.equal( other.m_pinch ) && (ecl_grid_compare( c_ptr() , other.c_ptr() , true , false , false )) && (m_minpvMode == other.getMinpvMode()));
-        if(m_minpvMode!=MinpvMode::ModeEnum::Inactive){
+        if (m_coord.size() != other.m_coord.size()) {
```

Comment:
Actually I think you can get away with just:
```
if (m_coord != other.m_coord)
     return false;
```

Reply:
thanks, done. Is this operator != (or ==) defined explicit in this scope, or is this part of std::vector container ?